app/package.json

```diff
@@ -2,12 +2,11 @@
   "name": "gem",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
   "scripts": {
     "dev": "parcel src/index.html",
     "dev:mock": "cross-env gem_API=mock yarn run dev",
     "mock": "cross-env gem_API=mock",
-    "watch": "parcel watch src/index.html --out-dir ../src/main/resources/static",
+    "watch": "parcel watch src/index.html --out-dir ../target/classes/static",
```

Comment:
Hmmm... does IntelliJ hook up nicely when it's routed to target? Later I try on Eclipse...

Reply:
Tested, working nicely in Eclispe too!

Reply:
I haven't been able to get the UI displayed (404 error) since the start. I'm still trying to get it to work but here are my findings so far:
1. build (mvn clean install) doesn't copy UI files into the BOOT-INF/static folder in the generated jar (possibly wrong folder should be app/dist instead of app/build in pom.xml)
2. build (mvn clean install) not generating dist folder (shouldn't we add "build": "parcel build src/index.html" to package.json?)
3. blank page is displayed instead of 404 error after making the changes, possibly due to blueprint scss errors after building UI using yarn build
4. spring profile doesn't seem to be working, always runs in default mode

Reply:
for (3), blank page is might be cause by gem_API=mock environment not set. Try 
- adding a .env with gem_API=mock
- delete app/.cache  and restart yarn watch
When rest api scaffolding is ready, we may not need gem_API=mock anymore.

for (1) and (2), I'll create a separate issues to work on them.
for (4):  not too familiar with spring-boot. can you create a new issue to describe this problem?